src/app/action-items/action-items.component.html

```diff
@@ -25,10 +25,14 @@
             <i class="fa fa-spinner fa-spin"></i>
         </div>
         <div class="tBody" *ngIf="!isLoadingActionItems">
+            <div *ngIf="showImageWhenNoActionItem" class="emptyListOfItem">
+                <img class="noItem" src="/assets/empty-inbox.svg">
```

Comment:
I would suggest to use `mat-icon` component and `IconRegistry` to display svg https://material.angular.io/components/icon/overview#svg-icons